src/clj/puppetlabs/services/ca/certificate_authority_service.clj

```diff
@@ -20,7 +19,10 @@
      (log/info "CA Service adding a ring handler")
      (add-ring-handler
        this
-      (compojure/context path [] (core/build-ring-handler settings puppet-version))))
+       (-> (core/web-routes settings)
+           (#(comidi/context path %))
```

Comment:
Not a big deal but where these lines are repeated per service's ring-handler, maybe worth pulling this out into a helper function at some point - either common to Puppet Server or in comidi itself?